db/migrate/20220824135136_add_mentor_to_submission_representations.rb

```diff
@@ -0,0 +1,12 @@
+class AddMentorToSubmissionRepresentations < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :submission_representations, :mentor, null: true, foreign_key: { to_table: :users }, if_not_exists: true
```

Comment:
I'd still rather this was `mentored_by` rather than `mentor` and that changing throughout. I find it much clearer.